internal/pkg/api/client.go

```diff
@@ -86,6 +86,9 @@ type ReenrollmentRequest struct {
 	// AttrReqs are requests for attributes to add to the certificate.
 	// Each attribute is added only if the requestor owns the attribute.
 	AttrReqs []*AttributeRequest `json:"attr_reqs,omitempty"`
+	// ReuseKey specifies whether to reuse the existing private key
+	// when reenrolling an identity.
+	ReuseKey bool `json:"reusekey"`
```

Comment:
I'm very confused why this field exists at this level and also within the CSRInfo. It looks like the implementation of `Reenroll` does not look at this but only the nested value.

Can you explain the intent?

Reply:
Good catch. This is leftover from my initial attempt. I'll remove it.

Reply:
Done